### DISCUSSION TITLE
Should we migrate "discussion" issues into GitHub Discussions?

### DISCUSSION THREAD

**Discussion by joshbuker** [cloudsecurityalliance/gsd-tools#144] [General]
Currently we have a lot of issues that are intended for community discussion / feedback ([8 issues at time of posting](https://github.com/cloudsecurityalliance/gsd-tools/issues?q=is%3Aissue+is%3Aopen+label%3Adiscussion)).

Would it make sense to migrate those issues into GitHub discussions? Anyone feel for or against the idea?

**Comment by kurtseifried** [comment]
I didn't even know discussions existed. This is clearly better, e.g. discussion/wide-ranging, and as action items pop up we turn them into issues.

**Comment by kurtseifried** [comment]
Yes, done.
